Makd.mak

```diff
@@ -113,6 +113,9 @@ RDMD ?= rdmd
 D_GC ?= cdgc
 export D_GC
 
+# Default documentation generator tool to use
+DDOC ?= hmod
```

Comment:
I think it might be nice to have too `DDOCFLAGS` or something, because if you change the tool, the target rules becomes pretty much useless (unless you are very lucky). Since harboredmod is also quite a particular tool in the way in which is invoked, I even wonder if this should be just called `HMOD` and `HMODFLAGS`.